postgres.DockerFile

```diff
@@ -1,3 +1,3 @@
-FROM postgres as database
+FROM postgres:15.0 as database
```

Comment:
You should probably prefer `postgres:15` over `postgres:15.0` since you won't get any security fixes this way (which are usually released as `15.x`)